code/game/objects/items/miscellaneous.dm

```diff
@@ -19,3 +19,86 @@
 	icon_state = "skub"
 	w_class = WEIGHT_CLASS_BULKY
 	attack_verb = list("skubbed")
+
+/obj/item/suspiciousphone
+	name = "suspicious phone"
+	desc = "This device raises pink levels to unknown highs."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "suspiciousphone"
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb = list("dumped")
+	var/dumped = FALSE
+	var/announcing = TRUE
+	var/calltype = CALLTYPE_NONE
+	var/datum/looping_sound/ring/soundloop
+	var/mob/bogdannof
+
+/obj/item/suspiciousphone/Initialize()
+	. = ..()
+	soundloop = new(list(src), FALSE, TRUE)
+	GLOB.suspicious_phones += src
+
+obj/item/suspiciousphone/Destroy()
+	. = ..()
+	QDEL_NULL(soundloop)
+	GLOB.suspicious_phones -= src
+
+/obj/item/suspiciousphone/equipped(mob/user, slot)
+	. = ..()
+	soundloop.output_atoms = list(user)
+	soundloop.direct = TRUE
```

Comment:
this way, no one else will be able to hear you making the call which blows your cover (which is good gameplay), are you sure you want this strength?

Reply:
its a niche item making it overly shit will mean its never used